pkg/sdk/go/responses.go

```diff
@@ -61,30 +61,29 @@ type MembersPage struct {
 	pageRes
 }
 
-type issueKeyRes struct {
+type KeyRes struct {
 	ID        string     `json:"id,omitempty"`
 	Value     string     `json:"value,omitempty"`
 	IssuedAt  time.Time  `json:"issued_at,omitempty"`
 	ExpiresAt *time.Time `json:"expires_at,omitempty"`
 }
 
-func (res issueKeyRes) Code() int {
+func (res KeyRes) Code() int {
 	return http.StatusCreated
 }
 
-func (res issueKeyRes) Headers() map[string]string {
+func (res KeyRes) Headers() map[string]string {
 	return map[string]string{}
 }
 
-func (res issueKeyRes) Empty() bool {
+func (res KeyRes) Empty() bool {
 	return res.Value == ""
 }
 
 type retrieveKeyRes struct {
 	ID        string     `json:"id,omitempty"`
 	IssuerID  string     `json:"issuer_id,omitempty"`
 	Subject   string     `json:"subject,omitempty"`
-	Type      uint32     `json:"type,omitempty"`
```

Comment:
Is this ok?

Reply:
@manuio I removed type because you can use only type 2 to Issue API Key.

Reply:
Then, why `keyReq{Type: APIKey, Duration: d}` have a type?